README.md

```diff
@@ -78,37 +78,71 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Let's say you want to train your `LinearRegression` model by your data `X`, `y`. After splitting the data into train/test segments you would have `X_train` and `y_train` as your training data. So you train your model.
```

Comment:
I think the used language is slangy! used the `Let's say` phrase multiple times.
I do not feel it, also the last sentence doesn't seem well.